site/src/components/MultiSelectCombobox/MultiSelectCombobox.tsx

```diff
@@ -203,9 +203,14 @@ export const MultiSelectCombobox = forwardRef<
 		const [open, setOpen] = useState(false);
 		const [onScrollbar, setOnScrollbar] = useState(false);
 		const [isLoading, setIsLoading] = useState(false);
-		const dropdownRef = useRef<HTMLDivElement>(null); // Added this
+		const dropdownRef = useRef<HTMLDivElement>(null);
 
-		const [selected, setSelected] = useState<Option[]>(value || []);
+		const [selected, setSelected] = useState<Option[]>(() => {
+			if (arrayDefaultOptions && arrayDefaultOptions.length > 0) {
+				return arrayDefaultOptions;
+			}
+			return [];
+		});
```

Comment:
looking at this again, it basically boils down to&hellip;

```suggestion
		const [selected, setSelected] = useState<Option[]>(arrayDefaultOptions ?? []);
```